Makefile

```diff
@@ -0,0 +1,35 @@
+.SILENT:
+.PHONY: build test
+
+## Colors
+COLOR_RESET   = \033[0m
+COLOR_INFO    = \033[32m
+COLOR_COMMENT = \033[33m
+
+## Help
+help:
+	printf "${COLOR_COMMENT}Usage:${COLOR_RESET}\n"
+	printf " make [target]\n\n"
+	printf "${COLOR_COMMENT}Available targets:${COLOR_RESET}\n"
+	awk '/^[a-zA-Z\-\_0-9\.@]+:/ { \
+		helpMessage = match(lastLine, /^## (.*)/); \
+		if (helpMessage) { \
+			helpCommand = substr($$1, 0, index($$1, ":")); \
+			helpMessage = substr(lastLine, RSTART + 3, RLENGTH); \
+			printf " ${COLOR_INFO}%-16s${COLOR_RESET} %s\n", helpCommand, helpMessage; \
+		} \
+	} \
+	{ lastLine = $$0 }' $(MAKEFILE_LIST)
+
+###############
+# COMMANDE #
+###############
+
+## initialize a git config to use hooks
+init-hooks:
+	git config --local core.hooksPath .githooks
+
+## to install app
+app-install:
+	npm install
+	make init-hooks
```

Comment:
Please add a blank line under this

Reply:
Do you test to commit with an other message like the regex ?
@Sbenziane  @Arnaud-Adon

Reply:
I just added an blank line and yes, I have tested commit with message that doesn't match with the regex.